src/main/java/roomescape/auth/AuthAdminInteceptor.java

```diff
@@ -0,0 +1,31 @@
+package roomescape.auth;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.HandlerInterceptor;
+import roomescape.infrastructure.JwtTokenUtil;
+import roomescape.member.Member;
+import roomescape.member.MemberService;
+
+@Component
+public class AuthAdminInteceptor implements HandlerInterceptor {
+    private MemberService memberService;
+    private JwtTokenUtil jwtTokenUtil;
```

Comment:
`final`로 처리하는 것이 더 좋아보여요!
[[Java] final 을 알아보자
](https://yeoonjae.tistory.com/entry/Java-final-%EC%9D%84-%EC%95%8C%EC%95%84%EB%B3%B4%EC%9E%90)

```suggestion
    private final MemberService memberService;
    private final JwtTokenUtil jwtTokenUtil;
```